data-prepper-plugins/translate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/translate/TranslateProcessor.java

```diff
@@ -44,18 +44,17 @@ public class TranslateProcessor extends AbstractProcessor<Record<Event>, Record<
     public TranslateProcessor(PluginMetrics pluginMetrics, final TranslateProcessorConfig translateProcessorConfig, final ExpressionEvaluator expressionEvaluator) {
         super(pluginMetrics);
         this.expressionEvaluator = expressionEvaluator;
-        mappingsConfig = translateProcessorConfig.getMappingsParameterConfigs();
-        mappingsConfig.forEach(MappingsParameterConfig::parseMappings);
-        parseFile(translateProcessorConfig.getFilePath());
-    }
-
-    private void parseFile(String filePath){
-        //todo
+        mappingsConfig = translateProcessorConfig.getCombinedParameterConfigs();
+        Optional.ofNullable(mappingsConfig)
+                .ifPresent(configs -> configs.forEach(MappingsParameterConfig::parseMappings));
```

Comment:
Are we going to stop the pipeline if only file_path option is present and the file is invalid?